menpo/image/base.py

```diff
@@ -1175,8 +1175,8 @@ def constrain_points_to_bounds(self, points):
         bounded_points[over_image] = shape[over_image]
         return bounded_points
 
-    def extract_patches(self, patch_centers, patch_size=(16, 16),
-                        sample_offsets=None, as_single_array=False):
+    def extract_patches(self, patch_centers, patch_shape=(16, 16),
```

Comment:
True... However, I think we should be consistent on the naming of this argument. So we need to choose either `patch_size` or `patch_shape` (I prefer it) and use it everywhere. I will make a PR in menpofit that will add a widget for patch-based appearance models visualization. So the PR can also include renaming all `patch_size` occurrences to `patch_shape`.